language/fixtures/super.rb

```diff
@@ -471,4 +471,26 @@ def foo(**)
     class C < A
     end
   end
+
+  module FromBasicObject
+    def __send__(name, *args, &block)
+      super
```

Comment:
@eregon - Yes it would work for this. The underlying bug with this wasn't about `__send__` per se, but it was the first example I picked.